.github/actions/generate-commit-list-v1/src/getRawCommitList.ts

```diff
@@ -10,15 +10,14 @@ import { GetRawCommitListParams } from './types'
 
 export default async function getRawCommitList({
   base,
-  head
+  head,
+  tag
 }: GetRawCommitListParams): Promise<string[]> {
   try {
-    const { stdout: rawCommitList } = await getExecOutput('git log', [
-      `origin/${base}..origin/${head}`,
-      '--oneline',
-      '--no-merges',
-      '--abbrev-commit'
-    ])
+    const subCommand = tag ? `${tag}..HEAD` : `origin/${base}..origin/${head}`
+    const { stdout: rawCommitList } = await getExecOutput(
+      `git log ${subCommand} --oneline --no-merges --abbrev-commit`
```

Comment:
This works, but worth nothing that the way `getExecOutput` works is that it will escape every character of this command. The command right now shouldn't be affected by escaping but it's worthing nothing for the future if we change the command that escaping would be problematic. In which case we need to go back to the `exect(command, [args])` style.

Reply:
for the future we will use `exec(commandLine: string, args?: string[], options?: ExecOptions)`